deploy/00_multisig.ts

```diff
@@ -5,21 +5,28 @@ import { DAY } from "../test-ts/utils";
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const { deploy } = hre.deployments;
 
-  const { deployer, multisigOwner0, multisigOwner1, multisigOwner2 } = await hre.getNamedAccounts();
-  const owners = [multisigOwner0, multisigOwner1, multisigOwner2];
+  // Get a list the named accounts based on network
+  const namedAccounts = await hre.getNamedAccounts();
+
+  const deployer = namedAccounts.deployer;
+  const minDelay = Number(process.env.TIME_LOCK_MIN_DELAY);
+  const delay = Number(process.env.TIME_LOCK_DELAY);
+
+  const multisigOwners = Object.values(namedAccounts).filter((val) => val !== deployer);
```

Comment:
Is this a good way of getting multisig owners? What if we later add other named accounts? What if the deployer is also a multisig owner?